packages/ui/src/atoms/dropdown/option.tsx

```diff
@@ -23,6 +23,7 @@ export const DropdownOption = ({
 }: IDropdownOption) => {
   return (
     <div
+      id={`option`}
```

Comment:
now all options will have id = "option", id should be unique for every dom element, so you need to pass custom id to options, where they are used

Reply:
@vitaliipustovyi ptal